### PR TITLE
Automated 4 UI test cases and fix 1 test failure

### PR DESCRIPTION
**Pull request by lihongyan1**
Depend on PR:
https://github.com/openshift/cucushift/pull/7909
Automated 4 test cases:
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-21263
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-24604
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-21124
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-24601
Fix one test failure for uncertain order:
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-24760
V3 job:
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/108195/console

### CODE REVIEW

**Comment by lihongyan1** [comment]
@juzhao @yapei Please help review.

File: lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -195,8 +195,23 @@ check_view_metrics:
...
+  action: click_link_with_text
 check_alert_description_on_rule_detail:
   params:
     text: This is an alert meant to ensure that the entire alerting pipeline is functional
     link_url: /monitoring/alerts
   action: check_link_and_text
+
+hide_alert_graph:
+  action: hide_show_alert_graph_button
+show_alert_graph:
```
Comment (juzhao) [inline lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml:211]: hide_alert_graph and show_alert_graph call for the same function hide_show_alert_graph_button
I don't think it is needed, but it makes sense that user can know the function by the name

File: lib/rules/web/admin_console/4.1/common_ui_elements.xyaml [hunk 2]
```diff
@@ -54,6 +54,11 @@ check_button_enabled:
   element:
     selector:
       xpath: //button[contains(., '<button_text>') and not(@disabled)]
+click_button_label:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/common_ui_elements.xyaml:57]: Is this action used?

File: lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml (hunk 1 above)
Comment (lihongyan1) [reply lib/rules/web/admin_console/4.2/monitoring_alerts.xyaml:211]: I suppose this way is better and make code in feature file is easy to read.

File: lib/rules/web/admin_console/4.1/common_ui_elements.xyaml (hunk 2 above)
Comment (lihongyan1) [reply lib/rules/web/admin_console/4.1/common_ui_elements.xyaml:57]: I deleted the changed

**Pull request merged**